WooCommerce/Classes/Authentication/WPComLogin/WPComPasswordLoginViewModel.swift

```diff
@@ -78,7 +78,7 @@ private extension WPComPasswordLoginViewModel {
         return email
             .lowercased()
             .trimmingCharacters(in: .whitespaces)
-            .md5Hash()
+            .sha256Hash()
```

Comment:
this diff is to fix the build failure from updating the WordPressUI library with this improvement https://github.com/wordpress-mobile/WordPressUI-iOS/pull/133.